### ISSUE TITLE
Unmatched parenthesis

### ISSUE THREAD

**Issue by ViacheslavP** [sympy/sympy#15400]
pycode.py contains unmatched parenthesis in 480. It seems to be working:

     479    def _print_im(self, expr):
     480        return "%s(%s)" % (self._module_format('numpy.imag'), self._print(expr.args[0]))

**Comment by bjodah** [comment]
I think you miscounted?

**Comment by ViacheslavP** [comment]
Sorry,  It's my first experience of bug reporting. I installed sympy from anaconda repo,  my program was unable to be executed,  the reason was in pycode.py, I added one parenthesis and it works. And then I had a thought that posting this here can help.

**Comment by bjodah** [comment]
No worries, my appreciate all feedback. The current version looks like this:
https://github.com/sympy/sympy/blob/ba60abbc7fe891dad9bd0bd0e7f90c8e15184de1/sympy/printing/pycode.py#L489

As far as I can tell it looks alright? It is possible that the bug is present in an earlier version of sympy. (mismatching parenthesis usually yield syntax errors thought, but perhaps the missing parenthesis was in the generated string?)

**Comment by ViacheslavP** [comment]
No, It not looks alright. Look at the same line below with numpy.real. It has to be a ) after 'numpy.imag'

**Comment by ViacheslavP** [comment]
Thank you for your respond!
